lib/ts/supertokens.ts

```diff
@@ -37,6 +38,15 @@ export default class SuperTokens {
     recipeModules: RecipeModule[];
 
     constructor(config: TypeInput) {
+        let inputValidation = validate(config, InputSchema);
+        if (inputValidation.errors.length > 0) {
+            let path = inputValidation.errors[0].path.join(".");
+            if (path !== "") {
+                path += " ";
+            }
+            let errorMessage = `${path}${inputValidation.errors[0].message}`;
+            throw new Error(`Config Schema Error: ${errorMessage}`);
```

Comment:
- No throwing of normal errors
- This code is duplicated in other places. Abstract it away and call that function